BIPs/2024-W23/BIP-613/inputs.json

```diff
@@ -0,0 +1,120 @@
+[
+  {
+    "chain_map": {
+      "mode": 34443,
+      "fraxtal": 252
+    },
+    "deployments": [
+      "weighted-pool-v4",
+      "composable-stable-pool-v6",
+      "composable-stable-pool-v5",
+      "20230409-erc4626-linear-pool-v4",
+      "20230409-gearbox-linear-pool-v2",
+      "yearn-linear-pool-v2",
+      "20230410-aave-linear-pool-v5",
```

Comment:
these five deployments dont exist on neither mode nor fraxtal

Reply:
The perm granter basically checks and adds whatever is there.  For that reason we just list all the pools that we have ever deployed/seen here and anything that exists gets permissions.  If you look at the end payload, none of them got permissions in the end/were ignored.  You can also look at the action run outputs for more details.